package.json

```diff
@@ -32,6 +32,7 @@
     "google-protobuf": "^3.6.1"
   },
   "devDependencies": {
+    "@bazel/typescript": "^0.20.3",
```

Comment:
you'll need to perform an `npm install` (using npm v6) to propagate this change through to the `package-lock.json` file.

Reply:
I might be looking at github ui wrongly, but I believe I did that already: 
https://github.com/improbable-eng/ts-protoc-gen/blob/9e831dedc3a359882f8ff4f84618a85dd4eba914/package-lock.json#L7

Reply:
Oh yes, so you have, sorry about that.

The build is failing because the `package-lock.json` is modified after the `npm install` invocation inside `generate.sh`.

Perhaps the version of npm being used differs between your local setup and on Travis?

Reply:
👍